packages/cbioportal-ts-api-client/src/generated/CBioPortalAPIInternal.ts

```diff
@@ -869,6 +869,8 @@ export type StructuralVariant = {
 
         'molecularProfileId': string
 
+        'namespaceColumns': {}
```

Comment:
can this type be specified?  i mean, what does that object look like?

Reply:
It cannot be specified: it contains a json object with custom columns that can be configured in the backend